app/assets/javascripts/utils/api.js

```diff
@@ -65,6 +65,23 @@ const API = {
       });
     });
   },
+  
+  fetchMoreRevisions(courseId, limit) {
```

Comment:
This should be named fetchCourseRevisions as well.

Reply:
Done! 👍